did/plugins/github.py

```diff
@@ -105,21 +105,30 @@ def commented_in_range(self,
                            login: str) -> list:
         valid_issues = []
         for issue in commented_issues:
-            comments = json.loads(
-                self.request(issue["comments_url"]).text
-                )
-            log.debug("Comments fetched for %s", issue["html_url"])
-            log.data(pretty(comments))
-            for comment in comments:
-                created_at = datetime.strptime(
-                    comment["created_at"],
-                    r"%Y-%m-%dT%H:%M:%SZ"
+            issue_since = since
+            valid_issue = False
+            while True:
+                url = (
+                    f"{issue['comments_url']}"
+                    f"?per_page={PER_PAGE}&since={issue_since.isoformat()}"
```

Comment:
is `since` parameter working here? If I got it right, GitHub's Comments API doesn't support a `since` query parameter on the comments endpoint.
The `since` parameter seems to be only available on the Events API, not the Issues/Comments API.